controllers/controller-articles.js

```diff
@@ -0,0 +1,14 @@
+const { selectArticle } = require("../models/model-articles")
+
+exports.getArticle = (req, res) => {
+
+    const { article_id } = req.params
+
+    return selectArticle(article_id).then((article) => {
+        res.status(200).send({article})
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
console.logs are the best for debugging but dont forget to remove them before pushing any code for review.